oidc-controller/api/core/oidc/issue_token_service.py

```diff
@@ -46,7 +47,9 @@ def get_claims(
             Claim(type="nonce", value=auth_session.request_parameters["nonce"])
         )
 
-        presentation_claims: Dict[str, Claim] = {}
+        presentation_claims: Dict[str, Claim] = {
+            "given_name": Claim("email", "email1@net.com")
```

Comment:
It looks like this made it's way through to the committed code

Reply:
Good catch, should be good to go now.